src/blocks/modal/render.js

```diff
@@ -96,20 +84,15 @@ const Render = ( props ) => {
 
 	const iconHTML = (
 		<div className='uagb-modal-trigger'>
-			{ '' !== icon && (
-				renderSVG( icon, setAttributes )
-			) }
+			{'' !== icon && renderSVG( icon, setAttributes ) }
 		</div>
 	);
 
-	const defaultedAlt = ( iconImage && iconImage?.alt ) ? iconImage?.alt : '';
+	const defaultedAlt = ( iconImage && iconImage )?.alt ? iconImage?.alt : '';
```

Comment:
```suggestion
	const defaultedAlt = iconImage?.alt ? iconImage.alt : '';
```
Please check this once